source/Octopus.Tentacle.Tests.Integration/ClientScriptExecutionCanBeCancelledWhenRetriesAreEnabled.cs

```diff
@@ -346,6 +346,7 @@ public async Task DuringStartScript_ForListeningTentacle_ThatIsRetryingTheRpc_An
                 .WithRetryDuration(TimeSpan.FromHours(1))
                 .WithTentacleServiceDecorator(new TentacleServiceDecoratorBuilder().RecordMethodUsages(tentacleConfigurationTestCase, out var scriptServiceRecordedUsages).Build())
                 .WithHalibutTimeoutsAndLimits(halibutTimeoutsAndLimits)
+                .WithPortForwarder(out var portForwarder)
```

Comment:
Let's validate that the port forwarder change is something worth keeping.

While investigating the flaky test, we were reminded that if multiple tests are running at the same time, then the port will become available again once we stop it.

When we call await clientAndTentacle.RunningTentacle.Stop(CancellationToken);, all it does is cancel the cancellation token and wait for the running tentacle task to finish. This would free up the port the tentacle is using.

I have kept this here, as it is a possible problem (unlikely, but possible).

It does make me wonder if this is something we should consider fixing in some for for all tests in general.